packages/buttons/src/elements/ToggleButton.tsx

```diff
@@ -11,9 +11,8 @@ import Button, { IButtonProps } from './Button';
 
 export interface IToggleButtonProps extends IButtonProps {
   /**
-   * Determine if the button is pressed or not. Use `'mixed'` to indicate
-   * whether the toggle controls other elements which do not share the same
-   * value.
+   * Determines if the button is pressed.
+   * Use `'mixed'` to indicate if the toggle controls other elements which don't share the same value.
```

Comment:
Paragraph formatting, as mentioned previously.

Reply:
Fixed in latest update